admin-dev/themes/new-theme/js/pages/product/product-map.js

```diff
@@ -116,5 +116,19 @@ export default {
     configuration: {
       fileManager: '.openfilemanager',
     },
+    photoswipe: {
+      element: '.pswp',
+    },
+    dzTemplate: '.dz-template',
+    dzPreview: '.dz-preview',
+    sortableContainer: '#product-images-dropzone',
+    sortableItems: 'div.dz-preview:not(.disabled)',
+    dropzoneContainer: '.dropzone-container',
+    checkbox: '.md-checkbox input',
+    shownTooltips: '.tooltip.show',
+    savedImage: (imageId) => `.dz-preview[data-id="${imageId}"]`,
+    savedImageElement: (imageId) => `.dz-preview[data-id="${imageId}"] img`,
```

Comment:
```suggestion
    dropzonePreviewImage: (imageId) => `.dz-preview[data-id="${imageId}"] img`,
```

Reply:
Same comment as above

Reply:
Well actually to me the `savedImageElement` IS the true image tag, and `savedImage` is the div container, that's why I suggested a renaming (although the one proposed is not good) Because it sounds like `savedImage` would select an image (as in `<img />`) not a div

So maybe `savedImageContainer` and `savedImage` then?

Reply:
I think it's better yes, gonna rename it